src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Figure implements Area, Output{
+    double a, b;
```

Comment:
Class-level variables 'a' and 'b' should be private and have descriptive names. Also, consider if these variables should be class-level or local to the methods that use them.